js-exercises/update-object/updateObject.js

```diff
@@ -0,0 +1,15 @@
+
+function updateObject(...args) {
+  if(args[0] === -1){
```

Comment:
to increase the readability `args[0]`, `args[1]`, `args[2]`can be assigned to variables like this respectively.
`const replacementIndex = arg[0]`, `const replacementValue = arg[0]`, `const array = arg[2]`.